_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -705,6 +705,11 @@ mfxStatus MFXVideoENCODE_Init(mfxSession session, mfxVideoParam *par)
         }
 
         mfxRes = session->m_pENCODE->Init(par);
+        if (MFX_WRN_PARTIAL_ACCELERATION == mfxRes)
+        {
+            // there is no sw callback
```

Comment:
There is mjpeg sw fallback. See line 227.

Please, provide a command line example where you see the issue. If some HW component returned this status - then this is bug in this component which we should fix rather than provide a workaround here.

Reply:
It's our internal test "beh_fei_encode_init" failure, tracked by MDP-47377.
It's AVCe/AVC FEI Encode returns this status, so let me reset the return value inside ImplementationAVC::Init().